pkg/api/mlflow/dao/repositories/param.go

```diff
@@ -32,48 +57,68 @@ func NewParamRepository(db *gorm.DB) *ParamRepository {
 
 // CreateBatch creates []models.Param entities in batch.
 func (r ParamRepository) CreateBatch(ctx context.Context, batchSize int, params []models.Param) error {
-	// try to create params in batch; error condition requires special handling
-	// to allow certain duplicates
-	if err := r.db.CreateInBatches(params, batchSize).Error; err != nil {
-		// remove duplicate rows and try again
-		dedupedParams, errRemovingMatches := r.removeExactMatches(ctx, params)
-		if errRemovingMatches != nil {
-			return eris.Wrap(errRemovingMatches, "error removing duplicates in batch")
+	if err := r.db.Transaction(func(tx *gorm.DB) error {
```

Comment:
Is this just tested via the integration tests? Is it not possible to unit test this method?

Reply:
I think not very useful to unit test since we are depending on database here